### ISSUE TITLE
No docisfy.min.esm.js generated

### ISSUE THREAD

**Issue by jcayzac** [docsifyjs/docsify#1435]
## Bug Report

No `docsify.esm.js` or `docsify.min.esm.js` is generated by the build script although they're mentioned at:

https://github.com/docsifyjs/docsify/blob/da9af18c4e4aa8b2c42773a01104f5a7c0a628a4/src/core/global-api.js#L12

Modules have been supported by all browsers for 2 years. Please give us a module 😄 

#### Steps to reproduce

```sh
curl https://unpkg.com/docsify@4.11.6/lib/docsify.esm.js
```

#### What is current behaviour

404 Not Found

#### What is the expected behaviour

200 OK

#### Other relevant information

- [x] Bug does still occur when all/other plugins are disabled?

- Your OS: 
- Node.js version: 
- npm/yarn version: 
- Browser version: 
- Docsify version: 4.11.6
- Docsify plugins: 

#### Please create a reproducible sandbox 

[![Edit 307qqv236](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/307qqv236)

**Comment by sy-records** [comment]
Should be added in v5? @trusktr 
```
// TODO This is deprecated, kept for backwards compatibility. Remove in next
// major release. We'll tell people to get everything from the DOCSIFY global
// when using the global build, but we'll highly recommend for them to import
// from the ESM build (f.e. lib/docsify.esm.js and lib/docsify.min.esm.js).
```

**Comment by trusktr** [comment]
@sy-records Nah, adding this is a non-breaking change. We could add it any time.

For v4, it would still create globals (but also export the globals for convenience of importing). For v5, there would be no globals.
